src/components/table-hoc/examples/TableHOCServerExamples.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+import {ReactVaporStore} from '../../../../docs/ReactVaporStore';
+import {IDispatch, ReduxConnect} from '../../../utils/ReduxUtils';
+import {IReactVaporTestState} from '../../../utils/TestUtils';
+import {TableHeaderWithSort} from '../TableHeaderWithSort';
+import {TableHOC} from '../TableHOC';
+import {TableHOCUtils} from '../TableHOCUtils';
+import {TableRowConnected} from '../TableRowConnected';
+import {tableWithBlankslate} from '../TableWithBlankslate';
+import {tableWithFilter} from '../TableWithFilter';
+import {tableWithPagination} from '../TableWithPagination';
+import {tableWithSort} from '../TableWithSort';
+import {IExampleRowData, TableHOCServerActions} from './TableHOCServerExampleReducer';
+
+const ServerTable = _.compose(
+    tableWithBlankslate({title: 'No data caused the table to be empty'}),
+    tableWithFilter({isServer: true}),
+    tableWithBlankslate({title: 'Filter caused the table to be empty'}),
+    tableWithSort({isServer: true}),
+    tableWithPagination({isServer: true, perPageNumbers: [3, 5, 10]}),
+)(TableHOC);
+
+interface TableHOCServerDispatchProps {
+    fetch: (compositeState: any) => void;

```

Comment:
Can `any` be typed here? At least put a generic.

Reply:
Yes, I'll create an interface for this